bench

```diff
@@ -68,9 +78,9 @@ def guarded_try_metric(benchmark: str, metric: str, function, unit="seconds", re
 def build_makefile():
     """ Builds VelCom with a makefile. """
     guarded_try_metric(
-        "frontend",
-        "build_time",
-        lambda: execute_program(["make", "-C", str(target_path()), "frontend"])
+       "frontend",
+       "build_time",
+       lambda: execute_program(["make", "-C", str(target_path()), "frontend"])
```

Comment:
Why did you change the indentation?

Reply:
By accident. I commented them out for testing